build.sh

```diff
@@ -2,7 +2,7 @@
 
 usage()
 {
-    echo "Usage: $0 [managed] [native] [BuildArch] [BuildType] [clean] [verbose] [clangx.y] [platform] [cross] [skiptests] [cmakeargs]"
+    echo "Usage: $0 [managed] [native] [BuildArch] [BuildType] [clean] [verbose] [clangx.y] [platform] [cross] [skiptests] [staticcurl]  [cmakeargs]"
```

Comment:
The option of having an option for each static library was first proposed by @mkborg in this issue https://github.com/dotnet/coreclr/issues/4436. I'm quite easy, and if you feel having one option that statically link all the library is better I will change the meaning of the parameter.